src/main/resources/templates/view_item.html

```diff
@@ -113,7 +113,7 @@ <h2>View an item</h2>
             <a type="date" th:text="*{expirationDate}" placeholder="YYYY-MM-DD" class="form-control"
                id="itemExpirationDate"></a>
           </div>
-
+<!--TODO: de revazut de ce nu mai merge sa vedem categoria-->
```

Comment:
let's keep this #TODO in english, please. also try not to keep so many #TODOs in the project, you can use GitHub issues

Reply:
I have made these changes everywhere. I made an update to commit (--amend). Please take a look. Thank you.